### PR TITLE
feat: add cap status to parsed event

### PR DESCRIPTION
**Pull request by antonanders**
minimal change to make filtering work. 
See https://github.com/jdemaeyer/brightsky/issues/166

### CODE REVIEW

**Comment by jdemaeyer** [comment]
Hi @antonanders, thanks a lot for the PR!

I'm attaching a test data file containing one of the test alerts, could you add that to `tests/data` and a small test checking that parsing it does indeed output a record with `record['status'] == 'test'`?

Also - and I realize this is super nitpicky :sweat_smile: - this repository uses single quotes for tokens / short strings, could you adjust that?

[Z_CAP_with_test.zip](https://github.com/jdemaeyer/dwdparse/files/15110803/Z_CAP_with_test.zip)

**Comment by antonanders** [comment]
Hi @jdemaeyer, 

thanks for pointing that out! Python is pretty new to me, so I hope, the changes are OK. :)

File: tests/test_parsers.py [hunk 1]
```diff
@@ -438,8 +438,12 @@ def test_radolan_parser(data_dir):
         [None, None, None, None, None],
     ]
 
-
 def test_cap_parser(data_dir):
+    test_cap_parser_actual_status(data_dir)
+    test_cap_parser_test_status(data_dir)
```
Comment (jdemaeyer) [inline tests/test_parsers.py]: `pytest` automatically picks up all functions prefixed with `test_` , so we don't need this wrapper function :)

File: tests/test_parsers.py [hunk 2]
```diff
@@ -474,11 +478,146 @@ def test_cap_parser(data_dir):
...
+        'response_type': 'prepare',
+        'urgency': 'immediate',
+        'severity': 'minor',
+        'certainty': 'likely',
+        'status': "test",
+        'effective': datetime.datetime(2024, 4, 23, 10, 21, tzinfo=datetime.timezone(datetime.timedelta(seconds=7200))), # noqa
+        'onset': datetime.datetime(2024, 4, 23, 10, 21, tzinfo=datetime.timezone(datetime.timedelta(seconds=7200))), # noqa
+        'expires': datetime.datetime(2024, 4, 23, 12, 0, tzinfo=datetime.timezone(datetime.timedelta(seconds=7200))), # noqa
+    }
+
```
Comment (jdemaeyer) [inline tests/test_parsers.py:489]: Our old test already covers parsing for all other fields, so probably just `assert records[0]['status'] == 'test'` is enough here?

**Comment by jdemaeyer** [comment]
No worries at all, I couldn't be more grateful that you're submitting a PR :) I've added two smaller comments in the hope you're not too annoyed yet, but after that we should be good to go!

**Comment by antonanders** [comment]
@jdemaeyer Thanks! 
Could you point me to the comments? I don't see them right now in the "File changed" tab or anywhere else.

**Comment by jdemaeyer** [comment]
Ah my bad, should be there now :)

**Comment by jdemaeyer** [comment]
Brilliant, merged with minor styling updates via 24759b1, thanks a lot! :) I'll add the changes to jdemaeyer/brightsky after resolving #3.

**Pull request closed without merging**